src/components/Panel/Wrap/PanelWrap.tsx

```diff
@@ -25,7 +25,7 @@ export function PanelWrap({
   const shouldCollapse = useMediaQuery(COLLAPSE_PANEL_QUERY);
 
   useEffect(() => {
-    if (shouldCollapse && !restrictAutoCollapsing) {
```

Comment:
All of this should be part of panel component 
![image](https://user-images.githubusercontent.com/6587585/190383675-79f234cc-e53b-40dc-aa63-3950d290341e.png)

Reply:
created a task #12405 to do that